package.json

```diff
@@ -1,18 +1,17 @@
 {
   "name": "pennbasics",
-  "version": "0.1.3",
+  "version": "1.0.0",
   "description": "Application built to allow University of Pennsylvania students to access information and services on one platform.",
-  "main": "webpack.config.js",
   "scripts": {
-    "start": "webpack -p && node src/server/index.js",
     "seedSpaces": "source env.sh && node src/server/database/seedSpacesInfo.js",
+    "seedFoodtrucks": "source env.sh && node src/server/database/scripts/seedFoodTrucksInfo.js",
+    "resetFoodtrucks": "source env.sh && node src/server/database/scripts/seedFoodTrucksInfo.js --reset",
     "test": "mocha --reporter spec",
-    "webpack": "webpack --watch",
-    "webpack-prod": "webpack -p",
-    "dev": "source env.sh && concurrently \"npm run webpack\" \"node src/server/index.js\"",
-    "lint": "eslint ./src/**/",
-    "lint-server": "eslint ./src/server",
-    "lint-fix": "eslint ./src/**/ --fix"
+    "lint": "eslint 'src/**/*.{ts,js,tsx,jsx}'",
+    "lint-fix": "eslint 'src/**/*.{ts,js,tsx,jsx}' --fix",
+    "dev": "source env.sh && nodemon",
+    "build": "next build",
+    "start": "source env.sh && ts-node --project tsconfig.server.json src/server/index.ts"
```

Comment:
If this is going to be used in prod, remove the `source env.sh`

Reply:
done